client/components/main/index.jsx

```diff
@@ -3,10 +3,10 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
```

Comment:
Since you're not making any functional changes to this file yet, I wouldn't touch it.